### PR TITLE
Completion of User Story 30 + User Stories 49-54

### PR DESCRIPTION
**Pull request by kristastadler**
Functionality being added:

Any item quantities in the order that were previously fulfilled have their quantities returned to their respective merchant's inventory for that item. (from User Story 30)

User Story 49 - created order show page for merchant employees details of an individual order containing one or more items from my store. Only display items from my store, not items from the order that are fulfilled by other stores.

User Story 50 - Adds functionality to order show page to fulfill each item from the order and reduces the items inventory once that order is fulfilled.

User Story 51 - Adds guard so items cannot be fulfilled if the merchant does not have enough stock in their inventory

User Story 52 - Created merchant index page for admin view

User Story 53 - Created user index page for admin view

User Story 54 - Created admin view of user profile/show page

### CODE REVIEW

File: app/controllers/merchant/orders_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class Merchant::OrdersController < Merchant::BaseController
+
+  def show
+    @merchant = Merchant.find(current_user.merchant_id)
```
Comment (mikez321) [inline app/controllers/merchant/orders_controller.rb:4]: This is the instance variable thing we fixed on the previous merge

File: app/models/item.rb [hunk 2]
```diff
@@ -51,4 +51,8 @@ def subtotal(order)
     price * order_qty_purchased(order)
   end
 
+  def order_status(order)  
+  item_orders.where(order_id: order).first.status
```
Comment (mikez321) [inline app/models/item.rb:55]: Needs a tab here.

File: spec/features/merchant_orders/show_spec.rb [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+        expect(page).to have_content("Inventory: 7")
+      end
+    end
+  end
+end
+
+
+
+
+# I see the item is now fulfilled
```
Comment (kmcgrevey) [inline spec/features/merchant_orders/show_spec.rb:127]: we can blip this out

File: spec/models/item_spec.rb [hunk 4]
```diff
@@ -7,6 +7,7 @@
     it { should validate_presence_of :price }
     it { should validate_presence_of :image }
     it { should validate_presence_of :inventory }
+
```
Comment (kmcgrevey) [inline spec/models/item_spec.rb:10]: this causes the Capybara error.  Recommended to skip is OK.

**Comment by kmcgrevey** [comment]
Reviewed together.  Approved merge.

**Pull request merged**